message_sender/views.py

```diff
@@ -230,6 +230,18 @@ def fire_delivery_hook(outbound):
         )
 
 
+def decr_message_count(message):
+    if not message.channel:
+        channel = Channel.objects.get(default=True)
+    else:
+        channel = Channel.objects.get(
+            channel_id=message.channel)
+
+    if channel.concurrency_limit != 0:
```

Comment:
I think we already do this check in the concurrency limiter, so probably no need to do it here.